docs/api/task-get-task-by-user-id.md

```diff
@@ -0,0 +1,58 @@
+---
+layout: page
+---
+
+# Get task by user ID
+
+Returns all task objects that are associated with a specific `user_id` parameter, if any exist.
+
+## URL
+
+```shell
+
+{server_url}/tasks/{user_id}
+```
+
+## Params
+
+| Parameter name | Type | Description |
+| -------------- | ------ | ------------ |
+| `user_id` | number | The ID of the user resource to which this task is assigned |
```

Comment:
Because this is a query, could the description be rephrased to reflect how the parameter affects the query?